app/views/users/show.html.erb

```diff
@@ -24,6 +25,11 @@
           <td scope="row"><%= event.date%></td>
           <td scope="row"><%= event.description%></td>
           <td> <%= link_to 'Edit', edit_event_path(event), :class=>"btn btn-outline-light" %></td>
+          <% if event.creator_id == @current_user.id %>
+            <td><%= button_to 'Destroy', event, method: :delete,data: { confirm: 'Are you sure?' }, class: 'btn btn-danger'%></td>
```

Comment:
Same as above comment

Reply:
I finished the modification and I made a new push